dags/resources/stages/parse/table_definitions/balancer/V1_ stkABPT_event_Transfer.json

```diff
@@ -0,0 +1,54 @@
+{
```

Comment:
@markusbkoch It looks like there is a space in the file name?

Reply:
Thank you, fixed